gradle.properties

```diff
@@ -2,17 +2,17 @@ org.gradle.jvmargs=-Xmx3072m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF
 dockerComposeStopContainers=true
 
 kotlinVersion=1.4.10
-okhttp3Version=4.9.+
-radarJerseyVersion=0.4.3
-radarCommonsVersion=0.13.+
-radarSchemasVersion=0.5.15
-radarOauthClientVersion=0.5.8
-jacksonVersion=2.11.+
-slf4jVersion=1.7.+
-logbackVersion=1.2.+
-grizzlyVersion=2.4.+
-lzfseVersion=0.1.+
-kafkaVersion=2.5.+
-confluentVersion=5.5.+
-junitVersion=5.+
-jedisVersion=3.3.+
+okhttp3Version=4.9.1
+radarJerseyVersion=0.6.2
+radarCommonsVersion=0.13.1
+radarSchemasVersion=0.6.0
+radarOauthClientVersion=0.7.1
+jacksonVersion=2.12.3
+slf4jVersion=1.7.30
+logbackVersion=1.2.3
+grizzlyVersion=3.0.0
+lzfseVersion=0.1.1
+kafkaVersion=2.7.0
+confluentVersion=5.5.2
```

Comment:
I don't think these versions match. Please check the confluent release notes which confluent version corresponds to which kafka version.